lib/space.ts

```diff
@@ -47,6 +47,11 @@ const spaceMethods: Array<keyof SpaceAPI> = [
   'getEntityScheduledActions',
 
   'signRequest',
+
+  'createTag',
+  'readTags',
```

Comment:
@damienxy I'm surprised to see `readTags` instead of `getTags` here given that we have `get{Users|Entries|...}` What am I missing?

Reply:
It is because the implementation uses the `tagsRepo` from the user interface, not directly the CMA.